data/qa_errors.json

```diff
@@ -1,36 +1,92 @@
 {
     "services": {
         "improveOSM": {
-            "errorTypes": {
-                "ow": {
-                    "icon": "fas-long-arrow-alt-right",
-                    "category": "routing"
```

Comment:
I guess the categories aren't used for anything?

Reply:
Yeah they were just a half implemented idea of my own with the thinking that issue styling could be made more simple once Osmose support was added (because there are so many issue types). Thankfully Osmose makes that easy already so I've just removed them.